src/bin/pgcopydb/env_utils.c

```diff
@@ -215,3 +217,83 @@ get_env_pgdata_or_exit(char *pgdata)
 			  "or from --pgdata");
 	exit(EXIT_CODE_BAD_ARGS);
 }
+
+
+/*
+ * get_env_using_parser checks if the environment variable exists and if it does
+ * it parses it and stores it in the target. If the environment variable is not
+ * set, the target is not modified. If the environment variable is set but the
+ * parsing fails, an error is logged and the errors counter is incremented.
+ */
+void
+get_env_using_parser(EnvParser *parser, int *errors)
```

Comment:
I think this should take an Array of EnvParser items and walk through it, maybe using the usual way we approach arrays with an EnvParserArray structure definition that has both the array and the count.

Reply:
Instead of changing this function, I introduced `get_env_using_parsers` later in the file. This new function iterates over the array elements and call `get_env_using_parser`.

This way I did not increase the indentation by one in this function. There is enough levels already here.